config/tracks/kicevo.ini

```diff
@@ -0,0 +1,430 @@
+[ABOUT]
+AUTHOR = ktulu77
+VERSION = 0.2
+DATE_RELEASE = 10/03/2021
+NOTES = Add rain, lights, weather and grass. Todo: remove some flower meshes in winter, add lights on towers, buildings and some vehicles.
+
+[INCLUDE]
+INCLUDE = common/all_conditions.ini
+
+[SOL]
+;HORIZON_OFFSET=0.0
+;DOME_SIZE=24000
+;SUN_DAWN=1
+;SUN_DUSK=1
+;FOG_LEVEL=0.85
+;FOG_BLEND=1.1
+;FOG_DISTANCE=1.0
+;SMOG_MORNING=0.1
+;SMOG_NOON=0.2
+;SMOG_EVENING=0.3
+;EXPOSURE_FIX=1
+
+[LIGHTING]
+SPECULAR_MULT = 1
+LIT_MULT = 1
+
+[DISTANT_GLOW]
+THRESHOLD = 25
+SHARED_THRESHOLD = 25
+
+[LIGHT_POLLUTION]
+RELATIVE_POSITION = 0.76, 0, 0.78
+DENSITY = 0.2
+RADIUS_KM = 0.7
+COLOR = 1.0, 0.55, 0.16
+ACTIVE = 1
+
+[WIND]
+DYNAMIC_FLAGS = NONE ; or AUTO not supported here
+
+[DEFAULTS]
+GrassMaterials=GRAS110?,Material #2, 140,Material #1
+TreesMaterials=Material #0
+BushesMaterials=
+2dSpectatorsMeshes=
+3dSpectatorsMeshes=publika
+RoadMaterials=RDAX107, 140, 98, RMBL138,145
+RoadPaintMaterials=?98?
+MetalMaterials=?MISC?, LowPolyCars, Material #251, ?ROADA72?, ?ROADA114?, ?Material #133?, trackVehicles, zastavakriva, WALL_TORANJ,Material #51,ROADA13,semaphore,ROADA118,ROADA52
+PlasticMaterials=twall_gume
+FabricMaterials=
+TentsMaterials=ROADA116,ROADA129,ROADA126,MISC
+GlassMaterials=
+RoughWoodMaterials=
+RoughRockMaterials=ROADA74
+SandMaterials=
+ConcreteBuildingMaterials=ROADA115,ROADA222,ROADA223,ROADA220, ROADA221,Material #134,ROADA115,zgradedst
+WoodBuildingMaterials=Material #122
+
+[SPECTATORS]
+MESHES = $2dSpectatorsMeshes, $3dSpectatorsMeshes
+
+[BOUNCED_LIGHT]
+SUPPORTED=1
+SURFACE_MATERIALS=$RoadMaterials, $GrassMaterials, $RoadPaintMaterials, $RoughWoodMaterials, $SandMaterials, $MetalMaterials, $RoughWoodMaterials
+
```

Comment:
You've defined surface materials but no occluding or object materials, making the lightmap fully white